### PR TITLE
[SCB-2082] add separated switches for metricsEndpoint and healthEndpoint

### PR DESCRIPTION
**Pull request by yhs0092**
- add config item `servicecomb.health.endpoint.enabled` as the switch of healthEndpoint
- add config item `servicecomb.metrics.endpoint.enabled` as the switch of metricsEndpoint
- the default value of these two item is `true`

### CODE REVIEW

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java [hunk 1]
```diff
@@ -43,13 +42,10 @@ public SlowInvocationLogger getSlowInvocationLogger() {
 
   @Override
   public void onBeforeProducerProvider(BootEvent event) {
-    if (!DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.metrics.endpoint.enabled", true).get()) {
+    if (!DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.metrics.endpoint.enabled", false).get()) {
```
Comment (liubao68) [inline metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java:45]: Maybe keep default value to true is better?

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/HealthBootListener.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import org.apache.servicecomb.metrics.core.publish.HealthCheckerRestPublisher;
+import org.springframework.stereotype.Component;
+
+import com.netflix.config.DynamicPropertyFactory;
+
+@Component
+public class HealthBootListener implements BootListener {
+  @Override
+  public void onBeforeProducerProvider(BootEvent event) {
+    if (!DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.health.endpoint.enabled", false).get()) {
```
Comment (liubao68) [inline metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/HealthBootListener.java:29]: Maybe keep default value to true is better?

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java (hunk 1 above)
Comment (yhs0092) [reply metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java:45]: Under the security sensitive circumstances, the users only want to expose their business REST interfaces. If we keep the default value to be `true`, they may expose these two REST interfaces by mistake.

Comment (liubao68) [reply metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java:45]: I think these two interfaces do not have any sensitive information. And if the application is secrured , the two interfaces will be secured too. I don't know why explode the two interface will cause security issue.

Comment (yhs0092) [reply metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/MetricsBootListener.java]: Done, please review

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/HealthBootListener.java (hunk 2 above)
Comment (yhs0092) [reply metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/HealthBootListener.java]: Done, please review

**Review by liubao68 — APPROVED** [review]

**Pull request merged**